File: app/src/main/java/se/gustavkarlsson/skylight/android/background/providers/impl/openweathermap/OpenWeatherMapService.java
package se.gustavkarlsson.skylight.android.background.providers.impl.openweathermap;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface OpenWeatherMapService {
	@GET("weather")
	Call<OpenWeatherMapWeather> get(@Query("lat") double latitude, @Query("lon") double longitude, @Query("mode") String mode, @Query("appid") String appId);
}

File: app/src/main/kotlin/se/gustavkarlsson/skylight/android/background/providers/impl/openweathermap/OpenWeatherMapService.kt
package se.gustavkarlsson.skylight.android.background.providers.impl.openweathermap

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface OpenWeatherMapService {
    @GET("weather")
    fun get(@Query("lat") latitude: Double, @Query("lon") longitude: Double, @Query("mode") mode: String, @Query("appid") appId: String): Call<OpenWeatherMapWeather>
}
